### ISSUE TITLE
Sign Installer & Libraries

### ISSUE THREAD

**Issue by BoBiene** [OpenSC/OpenSC#2396]
### Problem Description

Currently the MSI installer and the deployed files / libraries are not signed with a code-signing-certificate.

### Proposed Resolution

Use a free for open source code signing certificate (like https://about.signpath.io/open-source) and singn both libraries and msi packages

**Comment by Viajaz** [comment]
This is a problem where Windows Defender Application Control (WDAC) in a strict mode.

**Comment by frankmorgner** [comment]
duplicate of #1129

**Comment by Viajaz** [comment]
@frankmorgner They are technically different, #1129 is asking for a GPG signed list of hashes of released installer files whilst #2396 is asking for the MSI's and binaries to be signed (Authenticode) with a X.509 Code Signing certificate from a public CA. The solution in #1129 won't solve the problem of unsigned binaries (Executables, DLL's) in a use case like Windows Defender Application Control which has security controls to verify aforementioned binaries at runtime.

The two can co-exist and are separate technologies.

**Comment by speller** [comment]
Any updates on this issue? Tried to install the MSI and was blocked by the Defender app. Please don't ask people to disable it.
